lib/roo_on_rails/railties/routemaster.rb

```diff
@@ -0,0 +1,36 @@
+require 'roo_on_rails/config'
+
+module RooOnRails
+  module Railties
+    class Routemaster < Rails::Railtie
+      initializer 'roo_on_rails.routemaster' do
+        next unless Config.routemaster_enabled?
+
+        $stderr.puts 'initializer roo_on_rails.routemaster'
+
+        abort 'Aborting: ROUTEMASTER_URL & ROUTEMASTER_UUID are required' if bus_credentials_blank?
+
+        require 'routemaster/client'
+
+        ::Routemaster::Client.configure do |config|
+          config.url = routemaster_url
+          config.uuid = routemaster_uuid
+        end
+      end
+
+      private
+
+      def bus_credentials_blank?
+        routemaster_url.blank? && routemaster_uuid.blank?
```

Comment:
Shouldn't this be `routemaster_url.blank? || routemaster_uuid.blank?` ? (Assuming both are mandatory)